### PR TITLE
Add taxLocations to organization settings

### PR DESCRIPTION
**Pull request by ashkarpetin**

### CODE REVIEW

**Review by colinbird — DISMISSED** [review]

**Review by ninjasimon — DISMISSED** [review]

File: openapi/components/schemas/Organizations/TaxLocation.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+type: object
+required:
+  - city
+  - country
+  - postalCode
+properties:
```
Comment (VladimirBerdnik) [inline openapi/components/schemas/Organizations/TaxLocation.yaml:6]: Do you think we can add `street` parameter for more accurate tax calculation as promised by TaxJar?
image

File: openapi/components/schemas/Organizations/OrganizationSettings.yaml [hunk 2]
```diff
@@ -19,3 +19,11 @@ properties:
         description: The rate for flat tax calculation.
   kyc:
     $ref: ../KycSettings.yaml
+  taxLocations:
+    type: array
+    description: >-
+      Additional organization addresses where merchant may need to collect taxes using `taxjar` tax calculator.
```
Comment (noon-dawg) [inline openapi/components/schemas/Organizations/OrganizationSettings.yaml:25]:
```suggestion
      Additional organization addresses where a merchant may wish to collect taxes using the `taxjar` tax calculator.
```

File: openapi/components/schemas/Organizations/TaxLocation.yaml (hunk 1 above)
Comment (ashkarpetin) [reply openapi/components/schemas/Organizations/TaxLocation.yaml:6]: I won't be changing TaxJar integration in scope of tax locations, but I will add `address` field. Thanks

**Comment by ashkarpetin** [comment]
php please review

**Review by dara123 — APPROVED** [review]

**Review by colinbird — APPROVED** [review]

**Pull request merged**